example/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ set(CMAKE_CXX_STANDARD 14)
 if(NOT APPLE AND NOT MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 endif()
+if(APPLE)
+  set( OPENSSL_ROOT_DIR "/usr/local/opt/openssl")
+endif()
```

Comment:
Since the directory can be machine/user dependent, it can be provided on the command line when people invoke it (depending on where they installed their openssl) using cmake -DOPENSSL_ROOT_DIR=...Thus I'll remove it for now. :) Thank you.